style.less

```diff
@@ -162,3 +172,9 @@
         z-index: 105; // set to what plugin edittable defines
     }
 }
+
+// Workaround for sticky menubar in argon
+main {
```

Comment:
I'm not happy with this argon specific override here. Other templates might have a different structure with a main element. I think this should be moved to the argon template.